provider/testdata/data/baked/calendar/japanese_v1/und.rs.data

```diff
@@ -0,0 +1,14 @@
+::icu_calendar::provider::JapaneseErasV1 {
```

Comment:
Suggestion: You can use the proper path now (the one used for JSON): `baked/calendar/japanese@v1/und.rs.data`

Reply:
I can't because the mod file with the lookup function is also in this module